pkg/ui/src/views/reports/containers/enqueueRange/index.tsx

```diff
@@ -0,0 +1,238 @@
+                {Print.Timestamp(event.time)}
+              </td>
+              <td className="enqueue-range-table__cell">
+                <pre>{event.message}</pre>
+              </td>
+            </tr>
+          ))}
+        </tbody>
+      </table>
+    );
+  }
+
+  renderResponse() {
+    const { response } = this.state;
+
+    if (!response) {
+      return null;
+    }
+
+    return (
+      <div>
+        <h2>Enqueue Range Output</h2>
+        {response.details.map((details) => (
+          <div>
+            <h3>Node n{details.node_id}</h3>
+
+            {this.renderNodeResponse(details)}
+          </div>
+        ))}
+      </div>
+    );
+  }
+
+  renderError() {
+    const { error } = this.state;
+
+    if (!error) {
+      return null;
+    }
+
+    return (
+      <div>Error running EnqueueRange: { error.message }</div>
+    );
+  }
+
+  render() {
+    return (
+      <div>
+        <Helmet>
+          <title>Enqueue Range</title>
+        </Helmet>
+        <div className="content">
+          <section className="section">
+            <div className="form-container">
+              <h1 className="heading">Manually enqueue range in a replica queue</h1>
+              <br />
+              <form onSubmit={this.handleSubmit} className="form-internal" method="post">
+                Queue:{" "}
+                <select onChange={this.handleUpdateQueue}>
+                  {QUEUES.map((queue) => (
+                    <option key={queue} value={queue}>{queue}</option>
+                  ))}
+                </select>
+                <br />
+                <input
+                  type="number"
+                  name="rangeID"
+                  className="input-text"
+                  onChange={this.handleUpdateRangeID}
+                  value={this.state.rangeID}
+                  placeholder="RangeID"
+                />
+                <br />
+                <input
+                  type="number"
+                  name="nodeID"
+                  className="input-text"
+                  onChange={this.handleUpdateNodeID}
+                  value={this.state.nodeID}
+                  placeholder="NodeID (optional)"
+                />
+                <br />
+                <label>
+                  <input
+                    type="checkbox"
+                    checked={this.state.skipShouldQueue}
+                    name="skipShouldQueue"
+                    onChange={() => this.setState({ skipShouldQueue: !this.state.skipShouldQueue })}
+                  />
+                  SkipShouldQueue
+                </label>
+                <br />
+                <input
+                  type="submit"
+                  className="submit-button"
+                  value="Submit"
+                />
+              </form>
+            </div>
+          </section>
+        </div>
+        <div>
+          <section className="section">
```

Comment:
no need to wrap the `section` in an extra `div`... in fact, no need to wrap the outer container in a `div`, you could use `React.Fragment` to avoid creating extra markup elements.